### ISSUE TITLE
Add snippets to workflows

### ISSUE THREAD

**Issue by bioworkflows** [bioworkflows/bioworkflows-help#330]
* [ ] https://bioworkflows.com/public/413c4852ca8b4f2db9a1aa931598b70f.9999/  ARMOR
* [ ] https://bioworkflows.com/public/mcclintock.1/  Snakemake FASTQ Whole genome sequencing Sample ID Mapping ngs_te_mapper2 relocate2 RelocaTE2 Taxonomy TEMP TEMP2 trim-galore
* [ ] https://bioworkflows.com/public/oecophylla.1/  Snakemake Sequence analysis
* [ ] https://bioworkflows.com/public/invitrospi.1/  Snakemake
* [ ] https://bioworkflows.com/public/snakemake_assemble.1/  Snakemake MultiQC Bowtie 2 MEGAHIT Anvio multiqc-bcbio multiqc_sav anvio-minimal Skewer
* [ ] https://bioworkflows.com/public/nanopore-sv-analysis.1/  Snakemake
* [ ] https://bioworkflows.com/public/qadabra.1/  
* [ ] https://bioworkflows.com/public/grape.1/  GRAPE Ped-sim
* [ ] https://bioworkflows.com/public/mykrobe_tb_workflow.1/  Snakemake
* [ ] https://bioworkflows.com/public/dea_seurat.1/  Snakemake
* [ ] https://bioworkflows.com/public/scget.1/  Snakemake Cell biology scGET
* [ ] https://bioworkflows.com/public/nanopore-workflow.1/  Snakemake
* [ ] https://bioworkflows.com/public/scrnaseq_processing_seurat.1/  Snakemake
* [ ] https://bioworkflows.com/public/modr.1/  Snakemake
* [ ] https://bioworkflows.com/public/smsk_khmer_trinity.1/  Snakemake r-sleuth
* [ ] https://bioworkflows.com/public/nanite.1/  Snakemake
* [ ] https://bioworkflows.com/public/dea_limma.1/  Snakemake limma
* [ ] https://bioworkflows.com/public/smsk_trinotate.1/  Snakemake TransDecoder trinotate
* [ ] https://bioworkflows.com/public/river-dl.1/  PyTorch
* [ ] https://bioworkflows.com/public/on-rep-seq.1/  Snakemake
* [ ] https://bioworkflows.com/public/snakeatac.1/  Snakemake
* [ ] https://bioworkflows.com/public/omicsdiagnostics.1/  Snakemake wbuild
* [ ] https://bioworkflows.com/public/metavirs.1/  Snakemake
* [ ] https://bioworkflows.com/public/3d-beacons-client.1/  Snakemake
* [ ] https://bioworkflows.com/public/dna-seq-mtb.1/  Snakemake
* [ ] https://bioworkflows.com/public/twist_dna_hematology.1/  Snakemake Genomics
* [ ] https://bioworkflows.com/public/smsk_454.1/  Snakemake FastQC Quality control report Sample annotation RNA Transcriptome assembly TagDust2
* [ ] https://bioworkflows.com/public/cytopipe.1/  
* [ ] https://bioworkflows.com/public/mrcovid.1/  Snakemake
* [ ] https://bioworkflows.com/public/gkm-pipeline.2/  Snakemake ls-gkm Transcription factors and regulatory sites
* [ ] https://bioworkflows.com/public/mixscape_seurat.1/  Snakemake
* [ ] https://bioworkflows.com/public/cyte-seek.1/  Snakemake
* [ ] https://bioworkflows.com/public/metamorph.1/  Snakemake
* [ ] https://bioworkflows.com/public/basedmux.1/  porechop Filtlong
* [ ] https://bioworkflows.com/public/ngs-fastq-to-cram.1/  Snakemake
* [ ] https://bioworkflows.com/public/arraydynamics.1/  Snakemake
* [ ] https://bioworkflows.com/public/smsk_gatk_germline.1/  Snakemake
* [ ] https://bioworkflows.com/public/smsk_tuxedo2.1/  Snakemake
* [ ] https://bioworkflows.com/public/gkm-pipeline.1/  Snakemake Sequence analysis ls-gkm
* [ ] https://bioworkflows.com/public/boario.1/  
* [ ] https://bioworkflows.com/public/crisprseq_screens.1/  Snakemake
* [ ] https://bioworkflows.com/public/wg-genealogy-smk.1/  
* [ ] https://bioworkflows.com/public/sctensor-experiments.1/  Snakemake Data acquisition scTensor
* [ ] https://bioworkflows.com/public/scrnaseq_analysis_workflow.1/  Snakemake
* [ ] https://bioworkflows.com/public/smalldisco.1/  Snakemake
* [ ] https://bioworkflows.com/public/tswf.1/  
* [ ] https://bioworkflows.com/public/cell-seek.1/  Snakemake
* [ ] https://bioworkflows.com/public/mr-seek.1/  Snakemake Genetics
* [ ] https://bioworkflows.com/public/cvrcseq.1/  Snakemake
* [ ] https://bioworkflows.com/public/sequencing_artifacts.1/  Snakemake
* [ ] https://bioworkflows.com/public/data_analysis_fair_workflow.1/  Snakemake
* [ ] https://bioworkflows.com/public/snakemake-workshop.1/  Snakemake
* [ ] https://bioworkflows.com/public/mir-seek.1/  Snakemake
* [ ] https://bioworkflows.com/public/bam-msisensor.1/  Snakemake Variant calling genetic variants Data index microsatellite MSI msisensor MSIsensor
* [ ] https://bioworkflows.com/public/metagenomics-workflow.2/  Snakemake
* [ ] https://bioworkflows.com/public/metagenomics-workflow.1/  Snakemake metagenomics
* [ ] https://bioworkflows.com/public/ccbr_circrna_ampliconseq.1/  Snakemake
* [ ] https://bioworkflows.com/public/ff2.1/  Snakemake RNA-Seq
* [ ] https://bioworkflows.com/public/aisd_shapefile_integration.1/  Snakemake Data acquisition
* [ ] https://bioworkflows.com/public/dahak-flot.1/  Snakemake
* [ ] https://bioworkflows.com/public/workflow_sci-atac-seq.1/  Snakemake
* [ ] https://bioworkflows.com/public/kaiju-snake.1/  Snakemake
* [ ] https://bioworkflows.com/public/profiler.1/  Snakemake
* [ ] https://bioworkflows.com/public/10x-benchmark-rna.1/  Snakemake
* [ ] https://bioworkflows.com/public/circrnadetection.1/  Snakemake
* [ ] https://bioworkflows.com/public/snakemake-example.1/  Snakemake Sequence analysis
* [ ] https://bioworkflows.com/public/diva-wgs.1/  Snakemake Sequence analysis Whole genome sequencing
* [ ] https://bioworkflows.com/public/metagenome-read-mapping.1/  Sequence analysis Mason
* [ ] https://bioworkflows.com/public/sliding-window-testset.1/  metagenomics Mason
* [ ] https://bioworkflows.com/public/rattlersnp.1/  
* [ ] https://bioworkflows.com/public/goal-uab.1/  
* [ ] https://bioworkflows.com/public/mw-miallot2021.1/  Snakemake
* [ ] https://bioworkflows.com/public/rna-seq-analysis-pipeline.3/  
* [ ] https://bioworkflows.com/public/mechip2.1/  
* [ ] https://bioworkflows.com/public/rtx-kg2.1/  GDBChEMBL DisGeNET HMDB miRBase
* [ ] https://bioworkflows.com/public/rna-seq-analysis-pipeline.2/  
* [ ] https://bioworkflows.com/public/mex-pipeline.1/  Snakemake FastQC fastp Variant Effect Predictor (VEP)
* [ ] https://bioworkflows.com/public/shivercovid.1/  
* [ ] https://bioworkflows.com/public/human_chimpanzee_ipscs_chromatin_accessibility.1/  scATAC-seq
* [ ] https://bioworkflows.com/public/ccbr_rmats.1/  Snakemake IsoformSwitchAnalyzeR bioconductor-isoformswitchanalyzer
* [ ] https://bioworkflows.com/public/shiny-couscous.1/  Snakemake
* [ ] https://bioworkflows.com/public/rattlesnp.1/  
* [ ] https://bioworkflows.com/public/westac_parlaclarin_pipeline.1/  
* [ ] https://bioworkflows.com/public/admixtools.1/  
* [ ] https://bioworkflows.com/public/poa_genome.1/  
* [ ] https://bioworkflows.com/public/herv_tabulasapiens.1/  Snakemake
* [ ] https://bioworkflows.com/public/herv_scebv.1/  Snakemake
* [ ] https://bioworkflows.com/public/nciccr.1/  Snakemake
* [ ] https://bioworkflows.com/public/sctelescope.1/  Snakemake
* [ ] https://bioworkflows.com/public/differential-expression_workflow.1/  Snakemake
* [ ] https://bioworkflows.com/public/genome_assembly_tryouts.1/  
* [ ] https://bioworkflows.com/public/hula-heliorhodopsins.1/  Snakemake Sequence analysis
* [ ] https://bioworkflows.com/public/wgs_a2g_konigskobra.1/  Snakemake
* [ ] https://bioworkflows.com/public/ams_umc.1/  Snakemake
* [ ] https://bioworkflows.com/public/gdan-tmp-models.1/  
* [ ] https://bioworkflows.com/public/mirna.2/  Snakemake Sequence analysis
* [ ] https://bioworkflows.com/public/ccbr_ase.1/  Snakemake IsoformSwitchAnalyzeR bioconductor-isoformswitchanalyzer
* [ ] https://bioworkflows.com/public/ms-pipeline1.1/  
* [ ] https://bioworkflows.com/public/pipelord2_0.1/  Genomics
* [ ] https://bioworkflows.com/public/varfish-data-igsr.1/  Snakemake BCFtools tabix
* [ ] https://bioworkflows.com/public/scrna_r4.1/  Snakemake
* [ ] https://bioworkflows.com/public/snakemaketutorial.1/  Snakemake
* [ ] https://bioworkflows.com/public/wts_leukemia_konigskobra.1/  Snakemake
* [ ] https://bioworkflows.com/public/nugen-pilot-analysis.1/  Sequence analysis Variant calling Allegro
* [ ] https://bioworkflows.com/public/urchin-workflow.1/  
* [ ] https://bioworkflows.com/public/intro_data_analysis_bioinformatics.1/  Snakemake
* [ ] https://bioworkflows.com/public/glossy-couscous.1/  Snakemake
* [ ] https://bioworkflows.com/public/test.2/  Snakemake
* [ ] https://bioworkflows.com/public/bolt-lmm-gwas-pipeline.1/  Snakemake
* [ ] https://bioworkflows.com/public/rna-seq-1.1/  Snakemake
* [ ] https://bioworkflows.com/public/sn-mg-pipeline.1/  Snakemake MEGAHIT sourmash
* [ ] https://bioworkflows.com/public/datasources-smk.1/  
* [ ] https://bioworkflows.com/public/aquacrop-vegmon-analysis.1/  Snakemake
* [ ] https://bioworkflows.com/public/sstar-analysis.1/  Snakemake ms
* [ ] https://bioworkflows.com/public/autofj.1/  gene similarity analysis SigMod
* [ ] https://bioworkflows.com/public/dlscid.1/  Snakemake
* [ ] https://bioworkflows.com/public/snake_tree.1/  Snakemake
* [ ] https://bioworkflows.com/public/tranquil.1/  Snakemake mimseq
* [ ] https://bioworkflows.com/public/annotation_smk.1/  UniProt
* [ ] https://bioworkflows.com/public/snakemake_bammetrics_mosdepth.1/  Snakemake
* [ ] https://bioworkflows.com/public/quac.1/  Snakemake SAMtools BCFtools MultiQC Picard QualiMap mosdepth
* [ ] https://bioworkflows.com/public/rnaseq_pipeline.1/  Snakemake
* [ ] https://bioworkflows.com/public/ont-assembly-pipeline.1/  Snakemake BWA fastp Flye Medaka Filtlong
* [ ] https://bioworkflows.com/public/excercise_sars.1/  
* [ ] https://bioworkflows.com/public/haptools-paper.1/  Snakemake
* [ ] https://bioworkflows.com/public/artic-smk.1/  Snakemake MAFFT API (EBI) nanopore artic GUPPY
* [ ] https://bioworkflows.com/public/snake-dreem.1/  Snakemake
* [ ] https://bioworkflows.com/public/haptools-prs-sim.1/  Snakemake
* [ ] https://bioworkflows.com/public/c-integration-mapping.1/  Snakemake
* [ ] https://bioworkflows.com/public/kmer_workflow.1/  
* [ ] https://bioworkflows.com/public/mrna-seq-pipeline.1/  Snakemake
* [ ] https://bioworkflows.com/public/snakesplice.1/  Snakemake
* [ ] https://bioworkflows.com/public/test_snake.1/  Snakemake
* [ ] https://bioworkflows.com/public/pset.1/  Snakemake
* [ ] https://bioworkflows.com/public/fair-crcc-get-data.1/  Snakemake python-crypt4gh
* [ ] https://bioworkflows.com/public/scout-annotation.1/  
* [ ] https://bioworkflows.com/public/pyriksprot_tagger.1/  
* [ ] https://bioworkflows.com/public/ascent.1/  Snakemake IsoformSwitchAnalyzeR bioconductor-isoformswitchanalyzer
* [ ] https://bioworkflows.com/public/ervx.1/  Snakemake STAR JSON raw sequence reads Quality control report Gene expression Arriba FASTQ-illumina Sequence read processing
* [ ] https://bioworkflows.com/public/sv-jim.1/  Snakemake SAMtools BCFtools tabix Minimap2 sniffles cuteSV
* [ ] https://bioworkflows.com/public/vdblab-amplicon.1/  Snakemake
* [ ] https://bioworkflows.com/public/fold_tree.1/  Snakemake IQ-TREE
* [ ] https://bioworkflows.com/public/snakemake_falco.1/  Snakemake
* [ ] https://bioworkflows.com/public/giab-stratifications.1/  Snakemake
* [ ] https://bioworkflows.com/public/rna-seq-pipeline.1/  Snakemake
* [ ] https://bioworkflows.com/public/nanoseq-snakemake.1/  Snakemake SAMtools FastQC MultiQC BWA Sequence analysis
* [ ] https://bioworkflows.com/public/vdblab-shotgun.1/  Snakemake SAMtools FastQC Bowtie 2 seqkit kraken2 BBMap QUAST Krona SPAdes humann bracken ABRicate MetaPhlAn antiSMASH metaspades hmmer CoverM AMRFinderPlus RGI Gene ID MetaWRAP SortMeRNA Metagenomic sequencing Gene report Gene prediction antismash-lite dbCAN2 MetaErg MetaQUAST plasmidspades rnaspades cloudSPAdes
* [ ] https://bioworkflows.com/public/radseq.1/  SAMtools FastQC BCFtools MultiQC BEDTools BWA tabix fastp seqtk FreeBayes Nextflow cd-hit Bedops RADSeq
* [ ] https://bioworkflows.com/public/by-covid-wp5-t5-2-baseline-use-case.1/  WorkflowHub R
* [ ] https://bioworkflows.com/public/performance-evaluation-of-katdetectr-and-other-kat.1/  WorkflowHub katdetectr
* [ ] https://bioworkflows.com/public/load-ome-zarr-image-with-labels-from-a-public-s3-r.1/  WorkflowHub Electron microscopy APICURON autolog CITE-sort dhf hypeR idr-py ontoisobsolete scikit-learn WU-CRISPR YeastMine Zarr
* [ ] https://bioworkflows.com/public/validate-a-tool-against-idr-data-load-image-with-1.1/  WorkflowHub
* [ ] https://bioworkflows.com/public/java-compss-matrix-multiplication-out-of-core-usin.1/  WorkflowHub
* [ ] https://bioworkflows.com/public/permedcoe-single-drug-prediction.1/  WorkflowHub decoupleR MaBoSS CARNIVAL progeny PhysiCell
* [ ] https://bioworkflows.com/public/permedcoe-covid19-pilot-workflow-pycompss.1/  Snakemake Seurat WorkflowHub MaBoSS PhysiCell
* [ ] https://bioworkflows.com/public/permedcoe-covid19-pilot-workflow-rbbt.1/  Seurat WorkflowHub MaBoSS PhysiCell
* [ ] https://bioworkflows.com/public/katdetectr.1/  WorkflowHub Data visualisation katdetectr
* [ ] https://bioworkflows.com/public/drc_biochemical_toecbd.1/  WorkflowHub Data acquisition
* [ ] https://bioworkflows.com/public/backtrackbb-multi-band-array-detection-and-locatio.1/  WorkflowHub
* [ ] https://bioworkflows.com/public/lysozyme-in-water-compss-workflow.1/  WorkflowHub
* [ ] https://bioworkflows.com/public/seurat-scrna-seq.1/  WorkflowHub RNA-Seq Data Plot Filtering
* [ ] https://bioworkflows.com/public/janis-germline-variant-calling-workflow-gatk.1/  WorkflowHub janis
* [ ] https://bioworkflows.com/public/protein-md-setup-tutorial-using-bioexcel-buildin-2.1/  WorkflowHub BioBB-Wfs
* [ ] https://bioworkflows.com/public/scipion-workflow-for-cryo-electron-microscopy-of-s.1/  WorkflowHub
* [ ] https://bioworkflows.com/public/nf-core-rnaseq.1/  STAR Salmon Sequence analysis WorkflowHub HISAT2 RNA-Seq Nextflow RNA RNA nf-core-rnaseq
* [ ] https://bioworkflows.com/juok/mgnify-raw-reads-taxonomic-profiling-pipeline.1/  Biopython fastp seqtk WorkflowHub Krona Bwa-mem2 Genomics Nextflow Metagenomic operational taxonomic units (mOTUs) MAPseq easel Infernal

**Comment by bioworkflows** [comment]
bioworkflows.com/public/fair-crcc-get-data.1/ is hard due to comments for shell:
